test/unit-test/test_lib/test_in_out/test_reader.py

```diff
@@ -7,12 +7,12 @@
 from mock import patch
 import re
 
-current_directory = os.path.dirname(os.path.realpath(__file__))
 
 class ReaderTest(unittest.TestCase):
 
     def setUp(self):
-        self.log_data = util.load_from_disk(current_directory+ "/data/log_data")
+        self.current_directory = os.path.dirname(os.path.realpath(__file__))
```

Comment:
It would be better to have the class variables current_directory and channel_name created in setup() getting cleaned up in tearDown().

Reply:
Thanks, This is fixed now.